riscv-asm.md

```diff
@@ -377,6 +385,50 @@ for PIC as seen by objdump:
 			4: R_RISCV_PCREL_LO12_I	.L0
 ```
 
+When used with the `%gprel` relocation function,
+the `la` pseudo instruction
+is used to load symbol addresses relative to the `gp`,
+as in the compact code model:
+
+```assembly
+	la a0, %gprel(msg + 1)
```

Comment:
I don't like that `la` depends on -fPIC except suddenly when you throw `%gprel` at it. I'd expect this to use the GOT for -fPIC and GP directly otherwise. I feel you really want `lla` here; this model _can't_ work with preemptible symbols, but `la` does, so this is dangerous.

Reply:
Why was this marked as resolved? The issue still stands.

Reply:
Please, stand by as I upload the new patch.